packages/@glimmer/runtime/lib/environment.ts

```diff
@@ -255,7 +255,7 @@ export abstract class Environment {
   }
 
   begin() {
-    assert(!this._transaction, 'a glimmer transaction was begun, but one already exists. You may have a nested transaction');
+    assert(!this._transaction, 'A glimmer transaction was begun, but one already exists. You may have a nested transaction, possibly caused by an earlier runtime exception while rendering. Please check your stack trace.');
```

Comment:
What do you think about this small tweak to the last sentance:

> Please check your console for the stack trace of any prior exceptions.

Reply:
that's better, thanks. I'll update

Reply:
^ updated